tests/utils/PolygonUtilsTest.cpp

```diff
@@ -459,22 +478,25 @@ TEST_F(PolygonUtilsTest, RelativeHammingLineSquareDisjunct)
     ASSERT_EQ(PolygonUtils::relativeHammingDistance(test_squares, test_line), 1.0);
 }
 
-TEST_F(PolygonUtilsTest, DISABLED_RelativeHammingLineLine) //Disabled because this fails due to a bug in Clipper of testing points inside a line-polygon.
+TEST_F(PolygonUtilsTest,
+       DISABLED_RelativeHammingLineLine) // Disabled because this fails due to a bug in Clipper of testing points inside a line-polygon.
```

Comment:
I'd rather put the comment on a separate line than break the two parameters into different lines.